docs/api/heat/mediums.rst

```diff
@@ -3,6 +3,12 @@
 Material Thermal Specification
 -------------------------------
 
+To create simulation setups that can be used in different solvers we need to create material specifications that contain all relevant information for each of the solvers. Specifically, when performing coupled thermal and optic simulations, each material definition will contain up to three different characteristic:
```

Comment:
* comma after "used in different solvers".
* pluralize last word ("characteristics")
* in 1-2, comma before "such as"
* 1, 3: optic -> optical?